app/services/foreman_seeder.rb

```diff
@@ -34,27 +35,33 @@ def hash
   end
 
   def execute
-    self.class.is_seeding = true
-    begin
-      @seeds.each do |seed|
-        Rails.logger.info("Seeding #{seed}") unless Rails.env.test?
+    Rails.logger.info("Acquiring Advisory-Lock '#{ADVISORY_LOCK}' -> 0x#{(ADVISORY_LOCK.to_s.hash & 0x7FFFFFFFFFFFFFFF).to_s(16)} for seeding")
```

Comment:
Perhaps `Foreman::AdvisoryLockManager.generate_lock_id` should be made public so it can be used here as well?